includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -28,6 +28,7 @@ class AMP_Allowed_Tags_Generated {
 						'value_url' => array(
 							'allow_empty' => true,
 							'allow_relative' => true,
+							'valid_host' => true,
```

Comment:
This isn't going to work since this file is generated from the spec via [`amphtml-update.py`](https://github.com/Automattic/amp-wp/blob/develop/bin/amphtml-update.py), so your change here will be lost.